### DISCUSSION TITLE
GUI inaccessable

### DISCUSSION THREAD

**Discussion by deathtical** [AdguardTeam/AdGuardHome#6042] [Q&A] [answered]
I have been using AdGuard for over a year without issue but then my microsd card on my RPi4 died. I grabbed a new one, installed the latest Raspian, followed [these steps](https://github.com/AdguardTeam/AdGuardHome/wiki/Raspberry-Pi), after the install I was able to access the GUI for initial setup along with adding DNS and testing that filtering was working. I closed my browser and since then I have been unable to get back to the GUI. When I SSH to my pi and check the status of AdGuard, it says it is running.

@adguard:~/AdGuardHome $ sudo ./AdGuardHome -s status
2023/07/24 21:47:51 [info] AdGuard Home, version v0.107.34
2023/07/24 21:47:51 [info] service: control action: status
2023/07/24 21:47:51 [info] service: running
2023/07/24 21:47:51 [info] service: action status has been done successfully on linux-systemd

I tried the _AdGuardHome -s restart_ command which says it ran successfully with no change to GUI access. When trying to access the GUI I get (I have removed my IP address for security reasons):

Unable to connect

An error occurred during a connection to x.x.x.x:3000.

-     The site could be temporarily unavailable or too busy. Try again in a few moments.
-     If you are unable to load any pages, check your computer’s network connection.
-     If your computer or network is protected by a firewall or proxy, make sure that Firefox is permitted to access the web.

I have gone through this a few times with the same result.

1. Install Raspian
2. Install updates.
3. Reboot.
4. Follow AdGaurd instructions.
5. Connect to GUI for initial setup.
6. Close browser and never get GUI back.

I searched around and none of the solutions to similar problems (not the exact same) have helped here. My background is more with Windows and networking, so if you give me guidance for linux, please do not assume I will know what you mean by default (ie: please don't ask me to change a setting unless you can please provide the commands to do so). Or if I need to modify a config file, I know how to use nano. I just need to know what file, where it is located, and what to add, modify,  or comment out. Let me know if there is anything else I can provide to help you to help me.

Thanks

Just before I submitted this, I tried something that occurred to me and tried going to http://adguard and that works. So I guess I just need to know why the IP no longer works but the hostname does, and how can I get both to work again?

**Comment by ppfeufer** [comment, marked as answer]
As far as I know (didn't have to re-install AGH since I initially installed it), port 3000 is used for the initial setup process. After that, it's port 80 (default HTTP port).

As long as you don't use a reverse proxy to access your AGH instance (Apache2 or Nginx) you don't need to change this behaviour. But if you like, you can set this port in your `AdGuardHome.yaml` file in the `http` section (Here it's port 8100 for example):
```yaml
http:
  address: 0.0.0.0:8100
```

**Reply by deathtical** [reply]
Interesting. Is this somewhat "new"? Before my pi died, I always accessed it with port 3000. I guess the important thing is that just using the IP or hostname works now. Thanks for the clarification.
